library/std/src/f128.rs

```diff
@@ -898,10 +898,12 @@ impl f128 {
 
     /// Computes the four quadrant arctangent of `self` (`y`) and `other` (`x`) in radians.
     ///
-    /// * `x = 0`, `y = 0`: `0`
-    /// * `x >= 0`: `arctan(y/x)` -> `[-pi/2, pi/2]`
-    /// * `y >= 0`: `arctan(y/x) + pi` -> `(pi/2, pi]`
-    /// * `y < 0`: `arctan(y/x) - pi` -> `(-pi, -pi/2)`
+    /// * `x >= +0`, `y >= +0` -> `[+0, pi/2]`
+    /// * `x <= -0`, `y >= +0` -> `[pi/2, pi]`
+    /// * `x <= -0`, `y <= -0` -> `[-pi, -pi/2]`
+    /// * `x >= +0`, `y <= -0` -> `[-pi/2, -0]`
```

Comment:
```rust
println!("{}", f32::atan2(-0.0, 0.0)); // -0
println!("{}", f32::atan2(0.0, -0.0)); // 3.1415927
println!("{}", f32::atan2(0.0, 0.0)); // 0
println!("{}", f32::atan2(-0.0, -0.0)); // -3.1415927
```
...that's kinda fucked up huh.

I think placing the comment about negative versus positive zero *after* this table-ish-thing makes it harder to read. We should lead with the comment that the sign of 0.0 can affect the result, rather than being treated as equal.